src/dosarray_graphing_focus.sh

```diff
@@ -5,13 +5,26 @@
 #
 # Use of this source code is governed by the Apache 2.0 license; see LICENSE
 #
-# $ DOSARRAY_HIST_FOCUS=10 python ../generate_availability_chart.py '../../exp4/sl_60total_start10_last20_apache_event/c*.log' > availability.data
-# $ ../dosarray_graphing_focus.sh . '' 10 30
+# $ export DOSARRAY_VERSION="0.3"
+# $ DOSARRAY_NHIST_RESULT=1 python generate_availability_chart.py '../example_experiment/c*log' > availability_focus.data
+# $ ../dosarray_graphing_focus.sh -i availability_focus.data -i focus_graph.pdf
 #
-# FIXME input and output filenames are hardcoded
 
-DATA_DIR=$1
-TITLE=$2
+while getopts "i:o" opt; do
```

Comment:
Do the script(s) that call `dosarray_graphing_focus.sh` need updating, to pass the parameters? This doesn't show up in this commit.

Reply:
There are no scripts which use dosarray_graphing_focus.sh currently

Reply:
Thanks for checking -- and does `dosarray_graphing_focus.sh` show up in the documentation then? I think it was used in the paper draft for one of the diagrams.

Reply:
We haven't documented the use of this script. Although even I remembers having these graphs in the paper I reviewed with you

Reply:
Thanks for checking. Where would be a good place to document it, within the documentation we already have?